manifests/init.pp

```diff
@@ -32,6 +32,8 @@
 #       the scheme: "fact_name:fact_value".
 #   $puppet_run_reports
 #       Will send results from your puppet agent runs back to the datadog service.
+#   $manage_ruby
+#       Invoke the ruby class to manage ruby install
```

Comment:
I agree that `$manage_dogapi_gem` seems more fitting for what this switch enables.

Reply:
Agreed, the current name is a bit misleading.